### DISCUSSION TITLE
Crashing 5 seconds after logging in

### DISCUSSION THREAD

**Discussion by ToBeForgotten** [runelite/runelite#16674] [Q&A]
My runelite keeps crashing 3-5 seconds upon logging in. I can't find any information on fixing it anywhere

**Comment by Nightfirecat** [comment]
The issue tracker is not a support forum. Please join the [Discord](https://runelite.net/discord) for questions or support.

**Reply by DiscMeBruh** [reply]
also having issues with game crashing randomly

**Reply by Felanbird** [reply]
> also having issues with game crashing randomly

![image](https://github.com/runelite/runelite/assets/41973452/b57d26ec-0a02-4cb7-8282-f6b34958a5f9)

**Reply by DiscMeBruh** [reply]
tried that
